deploy/doc.Dockerfile

```diff
@@ -3,15 +3,19 @@
 # https://hub.docker.com/_/golang
 FROM golang:1.13 as builder
 
-WORKDIR app/
+WORKDIR /go/app/
```

Comment:
This change is following best-practices of using an absolute path instead of relative.